tests/apps/test_similarity.py

```diff
@@ -222,142 +223,224 @@ def test_event_cardinality(photons, max_count, modes, expected):
     assert similarity.event_cardinality(photons, max_count, modes) == expected
 
 
+class TestGetState:
+    """Tests for the function ``strawberryfields.apps.similarity._get_state.``"""
+
+    def test_loss(self, monkeypatch):
+        """Test if function correctly creates the SF program for lossy GBS."""
+        graph = nx.complete_graph(5)
+        mock_eng_run = mock.MagicMock()
+
+        with monkeypatch.context() as m:
+            m.setattr(sf.LocalEngine, "run", mock_eng_run)
+            similarity._get_state(graph, loss=0.5)
+            p_func = mock_eng_run.call_args[0][0]
+
+        assert isinstance(p_func.circuit[1].op, sf.ops.LossChannel)
+
+    def test_no_loss(self, monkeypatch):
+        """Test if function correctly creates the SF program for GBS without loss."""
+        graph = nx.complete_graph(5)
+        mock_eng_run = mock.MagicMock()
+
+        with monkeypatch.context() as m:
+            m.setattr(sf.LocalEngine, "run", mock_eng_run)
+            similarity._get_state(graph)
+            p_func = mock_eng_run.call_args[0][0]
+
+        assert not all([isinstance(op, sf.ops.LossChannel) for op in p_func.circuit])
+
+    def test_max_loss(self):
+        """Test if function samples from the vacuum when maximum loss is applied."""
+        dim = 5
+        graph = nx.complete_graph(dim)
+        State = similarity._get_state(graph, 0, 1)
```

Comment:
Also `State` to `state` here

Classes should be capitalized, but this is an _instance_ of a class.